### PR TITLE
Replace /policy/test with /check_policies in add edit mileage page

### PR DESCRIPTION
**Pull request by KirtiGautam**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=2eHRsgm)

### CODE REVIEW

**Review by Dimple16 — CHANGES_REQUESTED** [review]
important comments

File: src/app/fyle/add-edit-mileage/add-edit-mileage.page.ts [hunk 1]
```diff
@@ -1812,47 +1814,43 @@ export class AddEditMileagePage implements OnInit {
...
-        }
-        /* Adding number of attachements and sending in test call as tx_num_files
-         * If editing an expense with receipts, check for already uploaded receipts
-         */
         if (etxn.tx) {
-          policyETxn.tx.num_files = etxn.tx.num_files;
+          transactionCopy.num_files = etxn.tx.num_files;
+
+          // Check for receipts uploaded from mobile
+          if (etxn.dataUrls && etxn.dataUrls.length > 0) {
```
Comment (Dimple16) [inline src/app/fyle/add-edit-mileage/add-edit-mileage.page.ts:1833]: we don't require `num_files` logic for mileage expense no? 
we can remove it off entirely, please check if this is required and remove it for both mileage and per diem

**Review by Dimple16 — CHANGES_REQUESTED** [review]

File: src/app/fyle/add-edit-mileage/add-edit-mileage.page.ts [hunk 2]
```diff
@@ -1812,47 +1814,43 @@ export class AddEditMileagePage implements OnInit {
 
   checkPolicyViolation(etxn) {
     // Prepare etxn object with just tx and ou object required for test call
-    return from(this.authService.getEou()).pipe(
-      switchMap((currentEou) => {
+    this.mileageRates$.subscribe((rates) =>
+      console.log(this.getMileageByVehicleType(rates, etxn.tx.mileage_vehicle_type))
+    );
+    return from(this.mileageRates$).pipe(
```
[the remark this answers is not in the document]
Comment (KirtiGautam) [reply src/app/fyle/add-edit-mileage/add-edit-mileage.page.ts:1820]: `this.mileageRates$` is observable, that's why I have used the `from()` to subscribe and its result i.e. rates are returned. Do you mean I should directly use the subscribe on `this.mileageRates$`

[the remark this answers is not in the document]
Comment (Dimple16) [reply src/app/fyle/add-edit-mileage/add-edit-mileage.page.ts:1817]: no, you were previously setting the observable here which shouldn't be the case, you can use it here,

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**